ToGather/ToGather/Global/Service/OnboardingViewModel.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  LastOnboardingViewModel.swift
+//  ToGather
+//
+//  Created by Seungwon Choi on 2022/06/09.
+//
+
+import Foundation
+import Firebase
+import UIKit
+import SwiftUI
+
+class OnboardingViewModel: ObservableObject {
```

Comment:
```suggestion
final class OnboardingViewModel: ObservableObject {
```

더이상 상속되지 않는다는 명시를 해주는게 좋을 것 같아요 ~